testing/adios2/unit/TestChunkV.cpp

```diff
@@ -0,0 +1,146 @@
+/*
+ * Distributed under the OSI-approved Apache License, Version 2.0.  See
+ * accompanying file Copyright.txt for details.
+ */
+#include <cmath>
+#include <cstdint>
+#include <cstring>
+
+#include <iostream>
+#include <limits>
+#include <stdexcept>
+
+#include <adios2.h>
+#include <adios2/common/ADIOSTypes.h>
+#include <adios2/toolkit/format/buffer/chunk/ChunkV.h>
+
+#include <gtest/gtest.h>
+
+namespace adios2
+{
+namespace format
+{
+
+static void print_array(const std::string name, const uint8_t *ptr, const uint8_t size)
```

Comment:
This is why the windows builds complain (when you call the function the last argument is `size_t`)

Reply:
fixed